rematchTutorial.md

```diff
@@ -337,21 +338,21 @@ const model = {
 		async loadMoreTreeImages(payload, state){
 			//{{{
 			log.debug('to load images')
-			const verityState		= state.verity
-			if (verityState.isLoading || !verityState.moreTreeImagesAvailable){
+			const verifyState		= state.verify
```

Comment:
Yes, I think you can guess who is the bad guy with poor English did this.  😉 
By the way, is it easy to use IDE like vscode to refactor the renaming? Is any manual work needed?

Reply:
I'm sure there was a better way, but I just did a find and replace on instances of `verity`.